evi-typescript-example/README.md

```diff
@@ -48,7 +48,10 @@ export VITE_HUME_API_KEY=<YOUR_API_KEY>
 export VITE_HUME_CLIENT_SECRET=<YOUR_CLIENT_SECRET>
 ```
 
-You can make these environment variables persistent by adding them to the `.env` file in the root folder of the repo, which currently exists with placeholders. Once the API secrets have been added to the file, they will persist there and you do not have to set them as environment variables going forward. The `.env` file is listed in the `.gitignore` of the repository, so there is no risk of accidentally committing these secrets to a public repository. 
+You can make these environment variables persistent by adding them to a file named `.env` in the root folder of the repo.
+> There is an example file called `.env.example` with placeholder values, which you can simply rename to `.env`.
```

Comment:
Suggested edit:

> Refer to the [.env.example](<url-to-example-file>) file for which environment variables to include in your `.env` file.

Reply:
Thank you; agreed - I'll make that edit now and follow up with its commit.